### DISCUSSION TITLE
New website is live w/ app key generator!

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1467] [Announcements]
## 🚀 New website is live!

https://speedtest-tracker.dev/ is now live with all the links you'll ever need to navigate to the source code, documentation **AND generate an application key** without needing to use the CLI!

**Comment by mariushosting** [comment]
> ## 🚀 New website is live!
> https://speedtest-tracker.dev/ is now live with all the links you'll ever need to navigate to the source code, documentation **AND generate an application key** without needing to use the CLI!

Hello @alexjustesen the automatic APP KEY GENERATOR was removed? replaced?

**Reply by alexjustesen** [reply]
Removed for now with plans to come back later in the year with a new website and new features. I've updated the docs to reflect the new `APP_KEY` generation instruction.

On a personal note I have a lot travel for work coming up and I'm expecting another kid this year. It didn't make sense from a cost perspective to host the site anymore and essentially have no opportunity to do anything with it until the fall.

**Reply by mariushosting** [reply]
> Removed for now with plans to come back later in the year with a new website and new features. I've updated the docs to reflect the new `APP_KEY` generation instruction.
> 
> On a personal note I have a lot travel for work coming up and I'm expecting another kid this year. It didn't make sense from a cost perspective to host the site anymore and essentially have no opportunity to do anything with it until the fall.

Perfectly understandable Alex! Good Luck with everything!
